### PR TITLE
Various clean-ups and fixes

### PR DESCRIPTION
**Pull request by HCastano**
I've fixed some of the descriptions and names around the grants program, as well as
some typos here and there.

### CODE REVIEW

**Review by DoubleOTheven — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -52,27 +59,26 @@ The main acceptance criteria of any submitted project would be its impact on the
...
 - **Liquid staking** a liquid staking provider takes token deposits, stakes those tokens, and gives the depositor a receipt which is redeemable for the staked tokens.
 - **Aggregator** that unites separate decentralized protocols and aggregates liquidity from a variety of decentralized exchanges to facilitate cost-efficient transactions.
 
-**Ink! Cutting-edge Showcase**
+**Technical Showcase**
 
-This category implies developing full-stack application that clearly demonstrates the advantages of ink! over other smart contract languages(e.g. Solidity). In this category you should focus on opening the full potential of ink. The case doesn't necessarily needs to be business oriented. However it should clearly and visually demostrate superiority of ink. 
+This category implies developing full-stack application that clearly demonstrates the advantages of ink! over other smart contract languages (e.g. Solidity). In this category you should focus on opening the full potential of ink. The case doesn't necessarily needs to be business oriented. However it should clearly and visually demostrate superiority of the ink! language.
```
Comment (DoubleOTheven) [inline README.md:72]: you missed one: `opening the full potential of ink.` -> `...ink!.`

File: README.md [hunk 2]
```diff
@@ -52,27 +59,26 @@ The main acceptance criteria of any submitted project would be its impact on the
...
+- efficient gas usage
+- decreased smart contract size
+- improved developer experience
+- improved debugging experience
 
-**Ink! Infrastructure**
+**Infrastructure**
 
-On this stage ink! ecosystem lacks a lof of infrastructure. We welcome you to be innovative and submit your ideas. We will prefer ideas that add differentiation to Polkadot/ink! ecosystem, rather then tools that simply duplicate EVM. 
+On this stage ink! ecosystem lacks a lot of infrastructure. We welcome you to be innovative and submit your ideas. We will prefer ideas that add differentiation to Polkadot/ink! ecosystem, rather then tools that simply duplicate EVM.
```
Comment (DoubleOTheven) [inline README.md:81]: "On this stage..." sounds strange.

File: README.md [hunk 3]
```diff
@@ -52,27 +59,26 @@ The main acceptance criteria of any submitted project would be its impact on the
...
+This category implies developing full-stack application that clearly demonstrates the advantages of ink! over other smart contract languages (e.g. Solidity). In this category you should focus on opening the full potential of ink. The case doesn't necessarily needs to be business oriented. However it should clearly and visually demostrate superiority of the ink! language.
 You can use following aspects as indicators:
-- gas
-- smart contract size (find a use case where the size of smart contract will be smaller then solidity - application with complex calculations)
-- code writing experience (how comfortable to write code)
-- debugging experience
-
+- efficient gas usage
+- decreased smart contract size
+- improved developer experience
```
Comment (DoubleOTheven) [inline README.md:76]: How can a "full-stack application" improve developer and debugging experiences? I think these should move to "Infrastructure"

File: README.md (hunk 2 above)
Comment (DoubleOTheven) [reply README.md:81]: Also, I would not say "our infrastructure sucks", but maybe something like: 

```The goal of this track is to improve the ink! ecosystem's infrastructure and tooling.```

Comment (DoubleOTheven) [reply README.md:81]: `...add differentiation to Polkadot/ink! ecosystem,` -> `...add differentiation to the Polkadot/ink! ecosystem,`

File: README.md [hunk 4]
```diff
@@ -143,7 +153,7 @@ The curators consists of individuals, and are responsible for evaluating bounty
    2. In the newly created fork, create a copy of the application template ([`applications/application-template.md`](https://github.com/smart-contract-bounty/Ecosystem-Grants/blob/master/applications/application-template.md)). 
    3. Name the new file after your project: `project_name.md`.
    4. Fill out the template with the details of your project. The more information you provide, the faster the review. Make sure your deliverables present a similar level of detail.
-   5. Once you're done, create a pull request in **this repozitory**. The pull request should only contain _one new file_—the Markdown file you created from the template.
+   5. Once you're done, create a pull request in **this repozitory**. The pull request should only contain _one new file_ — the Markdown file you created from the template.
```
Comment (nicolad) [inline README.md:156]: `repository`

cc @HCastano

**Pull request merged**